.github/workflows/publish-sdk.yml

```diff
@@ -5,8 +5,8 @@ on:
     types: [published]
 
 env:
-  ORG_GRADLE_PROJECT_ossrhUsername: ${{ secrets.OSSRH_USERNAME }}
-  ORG_GRADLE_PROJECT_ossrhPassword: ${{ secrets.OSSRH_PASSWORD }}
+  ORG_GRADLE_PROJECT_ossrhUsername: ${{ secrets.MAVEN_CENTRAL_TOKEN_USERNAME }}
```

Comment:
Requires these secrets to be set on the repo.

Reply:
Did you have sufficient privileges to set them or do you need to pair?

Reply:
secrets are set, thank you